code/game/machinery/doors/poddoor.dm

```diff
@@ -15,6 +15,53 @@
 	resistance_flags = FIRE_PROOF
 	damage_deflection = 70
 	poddoor = TRUE
+	var/ertblast = FALSE
+	var/deconstruction = INTACT
+
+/obj/machinery/door/poddoor/multitool_act(mob/living/user)
+	var/change_id = input("Set the shutters/blast door/blast door controllers ID. It must be a number between 1 and 100.", "ID", id) as num|null
```

Comment:
Does this let you hijack existing shutter doors somehow? Like, can I stop the armory from shutting down just by multitooling the doors?

Reply:
Yep, I should probably edit it so you can only change it if the panel is open, my bad

Reply:
its done